release-notes/opensearch-security.release-notes-2.0.0.0-rc1.md

```diff
@@ -0,0 +1,24 @@
+## 2022-04-20 Version 2.0.0.0
+
+Compatible with OpenSearch 2.0.0-rc1
+
+### Enhancements
+
+* Incremented version to 2.0-rc1. ([#1764](https://github.com/opensearch-project/security/pull/#1764))
+* Renames securityconfig folder to config in bundle step and makes relevant changes ([#1749](https://github.com/opensearch-project/security/pull/#1749))
+* Add signal/wait model for TestAuditlogImpl ([#1758](https://github.com/opensearch-project/security/pull/#1758))
+* Remove TransportClient auth/auth ([#1701](https://github.com/opensearch-project/security/pull/#1701))
+* Introduce dfm_empty_overrides_all setting to enable role without dls/fls to override roles with dls/fls ([#1735](https://github.com/opensearch-project/security/pull/#1735))
+* Add depreciation notice to security tools ([#1756](https://github.com/opensearch-project/security/pull/#1756))
+* [Practice] Reverting changes ([#1754](https://github.com/opensearch-project/security/pull/#1754))
```

Comment:
These links are incorrect

Reply:
All links should be `.../pull/<PR_number>`

Reply:
Thanks, fixed